README.md

```diff
@@ -18,8 +18,17 @@ While working on backend stuff (Clojure), please keep a terminal open at *all* t
 
 `docker-compose run backend lein test-refresh`
 
+While working on frontend stuff (Javascript), please keep a terminal open at *all* times with the following command running:
+
+`docker-compose run frontend yarn test-refresh`
+
 This will give you instant feedback through the tests.
 
+If you don't want to use docker (and have Leiningen/Yarn installed), you can use the non-docker versions of the commands, which **must** be executed from within the respective directories (`backend`and `frontend` respectively):
+
+`lein test-refresh`
```

Comment:
You really had this already commited? xD

Reply:
Yes I did 🙃